### ISSUE TITLE
false error message for items which has no parent

### ISSUE THREAD

**Issue by wehrstedt** [wehrstedt/jsdoc-tsd#14]

**Comment by wehrstedt** [comment]
For item's which has no parent's you get the following error message:
```
Can't add member 'myTestClass' to parent item 'undefined'. Unsupported parent member type: 'undefined'. Insert this item as a top level declaration```
```
---------------
**Comment**
The output is correct, but the error message has to be suppressed if an item has no parent item

**Comment by wehrstedt** [comment]
solved with #5d1a53bce6135341cf1098ce6be5eccb84a38f81
